website/source/docs/commands/_general_options.html.md

```diff
@@ -0,0 +1,8 @@
+- `-address=<addr>`: The address of the Nomad server. Overrides the `NOMAD_ADDR`
+  environment variable if set. Defaults to `http://127.0.0.1:4646`.
+
+- `-region=<region>`: The region of the Nomad server to forward commands to.
+  Overrides the `NOMAD_REGION` environment variable if set. Defaults to the
+  Agent's local region.
+
+- `-no-color`: Disables colored command output.
```

Comment:
I didn't actually write these. I took these from the existing "general_options_usage" heredoc. The use of a Ruby function instead of a partial added about 3 seconds to the bulid.